CONTRIBUTING.md

```diff
@@ -52,61 +52,100 @@ To label the API endpoint as a beta in readme.io, you should make the following
 h1::after {\n content: \"BETA\";\n background-color: rgb(237, 117, 71);\n color: rgb(255, 255, 255);\n border-radius: 2rem;padding: 8px 13px 8px;\n white-space: nowrap;font-size:12px;\n}
 ```
 
+## OpenAPI files
+
+Please be notified that openAPI files slightly differ depending on where we use them.
+
+- **[production/readOnly-openAPI.json]** - Spec version 3.0.1 for all external viewers.
```

Comment:
Yes there is a reason :) You are reading `CONTRIBUTING.md`, file meant for contributors.
But imagine that there is a company like Y-COM that they want to get a valid openAPI file.
That would NOT be `reference/OpenAPI.json` because we use there `"type": "null"` that is illegal in v3.0.1. So I would sent Y-COM link to `production/readOnly-openAPI.json` because this is the only fully valid file.